defaults/manager_persistent_snapshots.yaml

```diff
@@ -1,18 +1,68 @@
 aws:
-  snapshots:
+  bucket: "manager-backup-tests-permanent-snapshots-us-east-1"
+  confirmation_stress_template: "cassandra-stress read cl=QUORUM n={num_of_rows} -schema 'keyspace={keyspace_name} replication(strategy=NetworkTopologyStrategy,replication_factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -mode cql3 native  -rate threads=50 -col 'size=FIXED(64) n=FIXED(16)' -pop seq={sequence_start}..{sequence_end}"
+  snapshots_sizes:
+    5:
+      number_of_rows: 5242880
+      expected_timeout: 900  # 15 minutes
+      snapshots:
+        'sm_20230702185929UTC':
+          keyspace_name: "5gb_sizetiered_2022_2"
+          scylla_version: "2022.2.9"
+          number_of_nodes: 5
+        'sm_20230702201949UTC':
+          keyspace_name: "5gb_sizetiered_2022_1"
+          scylla_version: "2022.1.7"
+          number_of_nodes: 5
+        'sm_20230702190638UTC':
+          keyspace_name: "5gb_sizetiered_5_2"
+          scylla_version: "5.2.3"
+          number_of_nodes: 5
```

Comment:
is it used anywhere?

Reply:
No, just in case this infor will ever be needed

Reply:
to have it comfortably available